barretenberg/cpp/scripts/analyze_client_ivc_bench.py

```diff
@@ -71,10 +71,35 @@
     print(f"{key:<{max_label_length}}{time_ms:>8.0f}  {time_ms/total_time_ms:>8.2%}")
 
 
-# Relations breakdown
-# Note: The timings here are off likely because the tracking is occuring in a hot loop but 
-# they should be meaningful relative to one another
-print('\nRelation contributions (times to be interpreted relatively):')
+# Extract a set of components from the benchmark data and display timings and relative percentages 
+def print_contributions(prefix, ivc_bench_json, bench_name, components):
```

Comment:
This was logic previously used only to process bench data for the Relations but I wanted to reuse it for commitments so I just made it a method